webclient/src/containers/Initialize/Initialize.i18n.json

```diff
@@ -0,0 +1,6 @@
+{
+  "InitializeContainer": {
+    "title": "DID YOU KNOW",
+    "subtitle": "<1>Cockatrice is run by volunteers</1><1>that love card games!</1>"
```

Comment:
@ebbit1q here is an example using strings that have DOM components interspersed. See the `Trans` component in the controller.